config/prow/build-cluster/README.md

```diff
@@ -29,3 +30,40 @@ Copy secrets from old cluster to build cluster:
 # Copy over secrets needed for jobs
 ./add-secrets.sh
 ```
+
+### Prow Clusters
+
+- [Prow control plane cluster(default prow cluster)](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1-f/prow?project=knative-tests) \[Only Google employees can access]
+  - Prow deployments, core configs, and plugins are hosted here
+  - Prow job configs are hosted in knative/test-infra/config/prow/jobs
+- [Build cluster](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1-f/knative-prow-build-cluster) \[Only Google employees can access]
+  - Build cluster deployments are hosted here
+  - Boskos resources are hosted in the boskos folder
+- [Trusted cluster](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1-a/prow-trusted) \[Only Google employees can access]
+  - This is a very basic cluster with important secrets
+
+### Manually Deploy
+
+Manual deployments are defined as in [Makefile](./Makefile), specifically:
+
+- `make -C prow/knative deploy`: deploys all yamls under [cluster](./cluster)
+- `make -C prow/knative deploy-build`: deploys all yamls under [cluster/build](./cluster/build)
+
+## Prow Secrets
```

Comment:
There is a section above secrets. Should that be merged here?

Reply:
Yes, let's also port them over. This aligns well with the longer term plan to also manage all the secrets in declarative yaml files. 
cc @upodroid